src/applications/simple-forms/21-0966/pages/thirdPartyPreparerFullName.js

```diff
@@ -10,11 +10,13 @@ export default {
   uiSchema: {
     ...titleUI(
       'Your name',
-      <p>
-        <b>Note:</b> If you’re a representative who is recognized by the VA to
-        make decisions or sign for the Veteran or surviving dependent, you can
-        sign this application for them.
-      </p>,
+      <span>
+        If you’re representing a Veteran, or their spouse or child, add your
+        name here. <br />
+        <br />
+        <b>Note:</b> You must be recognized by VA to make decisions or sign for
+        them.
+      </span>,
```

Comment:
My understanding is that semantic HTML would have us use `<p>`s instead of `<span>`s. This would also eliminate the need for `<br />`s. Does this work?
```
<p>
  If you're representing...
</p>
<p>
  <b>Note:</b> You must be recognized...
</p>
```
If it needs to be a single node, can we wrap it in `<></>`?

Reply:
Typically this is spot on advice. However, this text is going into the `<legend>`. `<legend>` tags can only take "[phrasing content](https://html.spec.whatwg.org/multipage/form-elements.html#the-legend-element)" which is headings, spans, line breaks, and a few other items, but `<p>` isn't one of them. Also, there's already a <span> that will wrap this second line of `titleUI` (An H3 wraps the first "Your name") when code is rendered in the browser.  

I agree that what I've done here is clunky. One way around it would be to remove the bold, and just make it one long string. And if that's better than what I've done here, then I'm okay with that.

Here's what the rendered code looks like:

```
<legend class="schemaform-block-title" id="root__title">

   <h3 class="vads-u-color--gray-dark vads-u-margin-top--0 vads-u-margin-bottom--3">
      Your name
   </h3>

   <span class="vads-u-margin-bottom--0 vads-u-font-family--sans vads-u-font-weight--normal vads-u-font-size--base vads-u-line-height--4 vads-u-display--block">
      <span>
         If you’re representing a Veteran, or their spouse or child, add your name here. 
         <br>
         <br>
         <b>Note:</b> You must be recognized by VA to make decisions or sign for them.
      </span>
   </span>

</legend>
```

Would it just be better to turn that text into a string, instead of trying to visually hack it?

Reply:
Ah! That makes sense to me. I think it's ok to leave as-is. I'll approve now!